src/components/Dialog.tsx

```diff
@@ -15,14 +15,29 @@ export function Dialog(props: {
   closeButtonRef?: React.Ref<HTMLButtonElement>;
   title: React.ReactNode;
 }) {
+  const islandRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!islandRef.current) {
+      return;
+    }
+
+    const buttons = islandRef.current.querySelectorAll("button");
```

Comment:
According to the specification of [querySelectorAll](https://www.w3.org/TR/selectors-api/#queryselectorall), the close button will always be first.

Reply:
Thanks for the suggestion. It has been updated. Since `[href]` selects the stylesheet tag, It was replaced by `a` tag.